tools/run-tests.py

```diff
@@ -130,33 +130,42 @@ def get_binary_path(bin_dir_path):
 
 def get_arguments():
     parser = argparse.ArgumentParser()
-    parser.add_argument('--toolchain', action='store', default='', help='Add toolchain file')
-    parser.add_argument('--buildoptions', action='store', default='',
+    parser.add_argument('--toolchain', metavar='FILE',
+                        help='Add toolchain file')
+    parser.add_argument('--buildoptions', metavar='LIST',
                         help='Add a comma separated list of extra build options to each test')
-    parser.add_argument('--skip-list', action='store', default='',
+    parser.add_argument('--skip-list', metavar='LIST',
                         help='Add a comma separated list of patterns of the excluded JS-tests')
-    parser.add_argument('--outdir', action='store', default=OUTPUT_DIR,
+    parser.add_argument('--outdir', metavar='DIR', default=OUTPUT_DIR,
                         help='Specify output directory (default: %(default)s)')
-    parser.add_argument('--check-signed-off', action='store_true', default=False,
-                        help='Run signed-off check')
-    parser.add_argument('--check-signed-off-tolerant', action='store_true', default=False,
-                        help='Run signed-off check in tolerant mode')
-    parser.add_argument('--check-signed-off-travis', action='store_true', default=False,
-                        help='Run signed-off check in tolerant mode if on Travis CI and not checking a pull request')
-    parser.add_argument('--check-cppcheck', action='store_true', default=False, help='Run cppcheck')
-    parser.add_argument('--check-doxygen', action='store_true', default=False, help='Run doxygen')
-    parser.add_argument('--check-pylint', action='store_true', default=False, help='Run pylint')
-    parser.add_argument('--check-vera', action='store_true', default=False, help='Run vera check')
-    parser.add_argument('--check-license', action='store_true', default=False, help='Run license check')
-    parser.add_argument('--check-magic-strings', action='store_true', default=False,
+    parser.add_argument('--check-signed-off', metavar='TYPE', choices=['strict', 'tolerant', 'travis'], nargs='?', const='strict',
```

Comment:
Pylint checker reports too long line (over 120 chars). In fact there are other places not related to this PR, but we can fix that in a later PR.